File: wednesday.py

class Day:
    pass


class Wednesday(Day):
    state = { } # class level attribute
    
    def __init__(self, task_list, quote_of_the_day, x):
        self.task_list = task_list
        self.quote = quote_of_the_day
        self.state["something"] = x


w1 = Wednesday(["complete News Site I", "eat some cereal"], "Life is a garden - Dig it!!", "something")

w2 = Wednesday(["take a nap"], "Sleep is good", "new thing")

print(w1.state["something"])

File: wednesday.js

class Day {
    constructor(quoteOfTheDay) {
        this.state = { }
        this.state.quote = quoteOfTheDay
    }
}

class Thursday extends Day {

}

class Wednesday extends Day {
    static value = { }
    
    state = { 

    } // instance attribute
    
    constructor(taskList, quoteOfTheDay, x) {
        super(quoteOfTheDay)
        
        this.state.taskList = taskList
        Wednesday.value.something = x
    }
}

let w1 = new Wednesday(["complete News Site I", "eat some cereal"], "Life is a garden - Dig it!!", "something")

let w2 = new Wednesday(["take a nap"], "Sleep is good", "new thing")

console.log(Wednesday.value.something)

